developer_manual/app_publishing_maintenance/maintainer.rst

```diff
@@ -0,0 +1,33 @@
+.. _app-maintainer:
+
+===========
+Maintainers
+===========
+
+.. sectionauthor:: Christoph Wurst <christoph@winzerhof-wurst.at>
+
+The role of a Nextcloud app maintainer is to oversee all processes around an app. The responsibilities are:
+
+* Triage bug reports
+
+  + Make sure reported bugs are reproducible, ask for more info if necessary
+  + Check for duplicates
+  + Close resolved tickets
+
+* Manage pull requests
+
+  + Ensure pull request receive a review in reasonable time
+  + Merge pull requests that pass the peer review and all continuous integration checks
+  + Trigger backports if applicable
+
+* Manage releases
+
+  + Ensure pre-releases are ready for deployment in pre-production of Nextcloud server
```

Comment:
link to https://docs.nextcloud.com/server/latest/developer_manual/app_publishing_maintenance/app_upgrade_guide/index.html